### PR TITLE
Add unit testing to eval code

### PR DESCRIPTION
**Pull request by yusuf1759**

### CODE REVIEW

File: docs/evaluation.md [hunk 1]
```diff
@@ -26,17 +31,17 @@ leaderboard (coming soon).
...
-`split.csv` with `system_id` and `split` columns mapping PLINDER systems to `train`, or `test`.
+`split.parquet` with, at a minimum, `system_id` and `split` columns mapping PLINDER systems to `train`, or `test`.
 
 ### Commands
 
 #### Write scores
 
 ```bash
-python src/plinder/eval/docking/write_scores.py --prediction_file predictions.csv --data_dir PLINDER_DATA_DIR --output_dir scores --num_processes 64
+python src/plinder/eval/docking/write_scores.py --prediction_file tests/test_data/eval/predictions.csv --data_dir tests/test_data/eval --output_dir /tmp --num_processes 64
```
Comment (Ninjani) [inline docs/evaluation.md:41]: replace with plinder_eval, but why are we writing into /tmp?

File: docs/evaluation.md [hunk 2]
```diff
@@ -69,6 +74,30 @@ For oligomeric complexes:
...
+lddt_pli_amd_ave                             0.85815               0.510695
+lddt_pli_amd_wave                            0.85815               0.510695
+scrmsd_ave                                  1.617184               3.665143
+scrmsd_wave                                 1.617184               3.665143
+rank                                               1                      1
+```
+
 #### Write test stratification data
 
 (This command will not need to be run by a user, the `test_set.parquet` and `val_set.parquet` file will be provided with the split release)
```
Comment (Ninjani) [inline docs/evaluation.md:103]: replace with plinder_stratify and also this doesn't use num_processes anymore

File: docs/evaluation.md [hunk 3]
```diff
@@ -81,12 +110,34 @@ Makes `test_data/test_set.parquet` which
 
 - Labels the maximum similarity of each test system to the training set across all the similarity metrics
 - Stratifies the test set based on training set similarity into `novel_pocket_pli`, `novel_pocket_ligand`, `novel_protein`, `novel_all`, and `not_novel`
```
Comment (Ninjani) [inline docs/evaluation.md:112]: Needs to be 

```
- Stratifies the test set based on training set similarity into `novel_pocket_pli`, `novel_ligand_pli`, `novel_protein`, `novel_ligand`, `novel_all` and `not_novel`
```
But these are not fully thought out yet imo, @OleinikovasV

File: tests/test_eval.py [hunk 4]
```diff
@@ -132,3 +132,33 @@ def test_multi_protein_single_ligand_scoring(system_1ai5, predicted_pose_1ai5):
...
+            f"{plinder_src}/tests/test_data/eval --output_dir "+\
+            f"{prediction_csv.parent}  --num_processes 65"
+    p1 = subprocess.Popen(
+            cmd1, shell=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE)
+    p1.communicate()
+
+    score_df = pd.read_parquet(f"{prediction_csv.parent}/scores.parquet")
+    assert np.allclose(score_df.scrmsd_wave.to_list(), [1.617184, 3.665143])
```
Comment (Ninjani) [inline tests/test_eval.py:151]: Seems from failing test that you may need to sort by reference_system_id here first

Comment (yusuf1759) [reply tests/test_eval.py:151]: At least 7 tests are still failing on my end.

Comment (tjduigna) [reply tests/test_eval.py:151]: ? There's only one test failing in CI, but making updates for the conversion to `plinder_eval` and `plinder_stratify`

**Review by tjduigna — APPROVED** [review]

**Pull request merged**